src/utils/types.ts

```diff
@@ -44,3 +45,9 @@ export type PublicUser = {
 
 export type User = inferQueryOutput<"user.me">;
 export type GeoJsonUsers = inferQueryOutput<"mapbox.geoJsonUsersList">;
+
+export type CarpoolAddress = {
+  place_name: string;
+  center: [longitude: number, latitude: number];
+};
+export type CarpoolFeature = Feature & CarpoolAddress;
```

Comment:
What does "&" do here exactly?

Reply:
CarpoolFeature conforms to both CarpoolAddress and the external Feature type. Essentially, it has the properties of both types.